kernel/dev/vga/graphics.cpp

```diff
@@ -97,9 +105,13 @@ void init(FramebufferInfo info)
     {
         map_kernel_page(VADDR(page), page);
     }
+    // Alloc the backbuffer
+    backbuffer = (malloc(height* pitch));
```

Comment:
`backbuffer = malloc(height * pitch);`

Reply:
so close lol

Reply:
i'ts Done

Reply:
You didn't remove the unnecessary parenthesis on the outside.

Reply:
I thought I had already removed it, sorry, in next time I'll pay more attention

Reply:
It's alright. 🙂